src/main/java/com/revature/domain/Person.java

```diff
@@ -0,0 +1,100 @@
+package com.revature.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="ie_person")
+public class Person {
+
+	@Id
+	@Column(name = "p_id")
+	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="perSeq")
+	@SequenceGenerator(allocationSize=1, name="perSeq", sequenceName="person_seq")
+	private int id;
+
+	@Column(name = "p_firstname", nullable = false)
+	private String firstName;
+	
+	@Column(name = "p_lastname", nullable = false)
+	private String lastName;
+	
+	public Person() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)